Shot1344[] descriptors
